src/stores/notifications/NotificationColor.ts

```diff
@@ -20,6 +20,7 @@ export enum NotificationColor {
     // TODO: Remove bold with notifications: https://github.com/vector-im/element-web/issues/14227
     Bold, // no badge, show as unread
     Grey, // unread notified messages
+    BoldRed, // no badge, show as unread and notify there is a mention
```

Comment:
I'm not sure I fully understand this, and adding fields here has implications beyond notification states: the room list needs to consider this case.

Reply:
The designs introduces a new state where a badge is displayed without the notification count but with a differentiator if a user has been mentioned or not 

![Screen Shot 2021-11-01 at 11 29 53](https://user-images.githubusercontent.com/769871/139664918-9020a9ef-848c-4ea3-a394-0131c9bdf071.png)

The consideration I took when adding this was to place it in the correct order as the enum will be translated to an auto increment numeric value.
The code in `NotificationState.ts` is never checking hardcoded numerical value but relies more on the order of the `NotificationColor` enum properties.

`hasUnreadCount` will return `false`, and `hasMentions` will return `true`.

Let me know if you can think of any other cases that are not covered here

Reply:
Right, that sort of state is meant to be handled by the UI code (ignore the count from the notification state) - we shouldn't need a new colour for that.